src/app/Http/Controllers/Api/QuizController.php

```diff
@@ -31,12 +37,15 @@ public function getQuiz($id)
     public function getQuizAttempts($id)
     {
         // Implement logic to fetch quiz attempts
-        // TODO: attempts related to users, will be a new task in the sprint 4
+
+        $attempts = DB::table('attempt_quizzes')
+            ->where('quiz_id', $id)
+            ->get();
```

Comment:
To my understanding, this can be rewritten as:
```php
$attempts = AttemptQuiz::where('quiz_id', $ $id)->get();
```

Ideally we should keep `DB:table(...)` queries to a minimum if it can be avoided.